### ISSUE TITLE
How to get 10-fold split lists for CALFW and CPLFW?

### ISSUE THREAD

**Issue by wandering007** [TreB1eN/InsightFace_Pytorch#108]
Hi, thanks for providing this awesome project.

After I downloaded CALFW and CPLFW datasets from the official websites, I found that there are no off-the-shelf 10-fold cross-validation lists among the files. 

I want to ask that how the project process and evaluate CALFW and CPLFW datasets. 
It would be appreciated if you could provide me the 10-fold split list files. Thanks a lot!

**Comment by hjy1312** [comment]
For CPLFW,“Based on it, our
CPLFW dataset has been divided into 10 separate folds using the same identities contained in the LFW 10 folds.”

**Comment by wandering007** [comment]
@hjy1312 Thanks! I'll try it.

**Comment by wandering007** [comment]
@hjy1312 Hi, I have split the 10 folds correctly after matching identities with LFW dataset. But the results on CALFW and CPLFW are quite low (LFW: 99.7%, CALFW: ~70%, CPLFW: ~65%). Have you been ever faced with this issue before? Thanks!

**Comment by hjy1312** [comment]
@wandering007 Compared with the LFW pair list file, I suppose you can directly split the pair list into 10 parts in sequence. But I'm not sure whether the data should be divided in this way.

**Comment by wandering007** [comment]
> @wandering007 Compared with the LFW pair list file, I suppose you can directly split the pair list into 10 parts in sequence. But I'm not sure whether the data should be divided in this way.

Yes, quite like this. I am quite sure that the 10-fold split problem is solved. But I still get low results, which makes me very confused.

**Comment by hjy1312** [comment]
@wandering007 Maybe there're other factors influencing the experimental results. I test my model with the directly split 10 folds data, the accuracy is about 82.6%. And when I test the model with randomly selected 10 folds according to  [](https://github.com/egcode/pytorch-face/blob/ab4d058075e4a282273c6a4a63e5ddfaf03c92cb/validate.py), the accuracy is 80.667%.
